MonitorizareVot/UIColor+Presets.swift

```diff
@@ -1,45 +1,20 @@
 //  Created by Code4Romania
 
-import Foundation
 import UIKit
 
 extension UIColor {
     
-    static let navigationBarBackground = UIColor(hexCode: 0xFFCC00)
-    static let navigationBarTint = UIColor(hexCode: 0x333E48)
+    static var colorSchema: ColorSchema.Type {
+        // UserInterfaceStyle is available from iOS 12, but dark mode is only available from iOS 13.
+        if #available(iOS 13.0, *) {
+            return UIApplication.shared.keyWindow?.rootViewController?.traitCollection.userInterfaceStyle == .light ?
+                StandardColorSchema.self :
+                DarkModeColorSchema.self
+        } else {
+            return StandardColorSchema.self
+        }
+    }
```

Comment:
Single source of truth to decide what colour schema to be used. Should easily be extensible to support any number of future custom schemas.